bibtex-actions.el

```diff
@@ -187,6 +187,19 @@ key associated with each one."
 ;;  when this PR is merged:
 ;;    https://github.com/tmalsburg/helm-bibtex/pull/367
 
+(defvar bibtex-actions--candidates-cache nil
+  "Store the candidates list.")
+
+(defun bibtex-actions--get-candidates ()
+    "Load bib files and cache them."
+    (if (not bibtex-actions--candidates-cache)
+     (setq bibtex-actions--candidates-cache (bibtex-actions--format-candidates)))
+    bibtex-actions--candidates-cache)
+
+(defun bibtex-actions--clear-candidates-cache ()
```

Comment:
I'm a little unsure on this. Do we want this function, or should it rather be a "reload-candidates-cache" function?